hello_test.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+int main()
+{
+
```

Comment:
add few some space or intendation

Reply:
thanks for your help.
i added same thing as you suggested.

Regards
Jagadeesh P.